### PR TITLE
Prints all products from the category

### PR DESCRIPTION
**Pull request by antonmaskalik**

### CODE REVIEW

File: OnlineStore/Store.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            foreach (Category c in Categories)
+            {
+                if (c.GetType().Name == "BikeCategory")
+                {
+                    Console.WriteLine("Category bikes:");
+                    c.PrintProducts();
+                    Console.WriteLine(new string('-', 60));
+                }
+                if (c.GetType().Name == "MilkCategory")
+                {
```
Comment (MarinaGotka) [inline OnlineStore/Store.cs:25]: Should be found much easier way to print category name before the product list

File: OnlineStore/StoreHelpers/StoreHelper.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            GetCategories();
+            foreach (var category in store.Categories)
+            {
+                int count = random.Next(3, 15);
+
+                for (int i = 1; i <= count; i++)
+                {
+                    Product product = new Product();
+
+                    product.Rate = Math.Round(random.NextDouble() + random.Next(2, 9), 1);
```
Comment (MarinaGotka) [inline OnlineStore/StoreHelpers/StoreHelper.cs:44]: Replace please Property usage with constructor invocation:
Product product = new Product(category, name, price, rate);

**Review by MarinaGotka — CHANGES_REQUESTED** [review]

File: OnlineStore/StoreHelpers/StoreHelper.cs [hunk 3]
```diff
@@ -39,12 +39,12 @@ public void FillStore()
 
                 for (int i = 1; i <= count; i++)
                 {
-                    Product product = new Product();
+                    double rate = Math.Round(random.NextDouble() + random.Next(2, 9), 1);
+                    Category newcategory = category;
```
Comment (MarinaGotka) [inline OnlineStore/StoreHelpers/StoreHelper.cs:43]: This variable is not needed at all

**Review by MarinaGotka — CHANGES_REQUESTED** [review]

File: OnlineStore/StoreHelpers/StoreHelper.cs (hunk 2 above)
Comment (antonmaskalik) [reply OnlineStore/StoreHelpers/StoreHelper.cs]: Done

File: OnlineStore/StoreHelpers/StoreHelper.cs (hunk 3 above)
Comment (antonmaskalik) [reply OnlineStore/StoreHelpers/StoreHelper.cs:43]: Done

**Review by MarinaGotka — APPROVED** [review]

**Pull request merged**